upload/admin/model/extension/shipping/zasilkovna_orders.php

```diff
@@ -133,6 +133,8 @@ private function createFilterConditions(array $filterData) {
 			$sqlConditions[] = ' `o`.`order_status_id` IN (' . $orderStatusesString . ') ';
 		}
 
+		$sqlConditions[] = ' `o`.`order_status_id` > 0 ';
```

Comment:
Nebylo by hezčí, pokud by se podmínka 'order_status_id` > 0' aplikovala jen v případě,  že ve výsledném SQL dotazu nebude už '`order_status_id` IN ()' ?